src/Enum/ValueTypeEnum.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\SdkContracts\Enum;
+
+final class ValueTypeEnum
+{
+    /**
+     * @var string
+     */
+    public const TYPE_BOOLEAN = 'boolean';
```

Comment:
Inconsistent
Why int, but not bool?

Reply:
Seems like it comes from gettype(). changed to `integer`